app/models/indexing/scheduled_reindex_job.rb

```diff
@@ -5,7 +5,7 @@ def self.perform(reindex_type)
               when 'main'
                 %w(Pseud Tag Work Bookmark Series ExternalWork)
               when 'background'
-                %w(Work Bookmark Pseud Series ExternalWork)
```

Comment:
Do we ever want to have a class in one but not both of "main" and "background", or can we do `when "main", "background"`?

Reply:
Not that I can find, so I'll do that.